source/solvers/gls_navier_stokes.cc

```diff
@@ -191,25 +191,29 @@ template <int dim>
 void
 GLSNavierStokesSolver<dim>::update_boundary_conditions()
 {
-  double time = this->simulation_control->get_current_time();
-  for (unsigned int i_bc = 0;
-       i_bc < this->simulation_parameters.boundary_conditions.size;
-       ++i_bc)
+  if (this->simulation_parameters.boundary_conditions.time_dependent)
```

Comment:
Is this better (more efficient) than checking if false + return like in the auxiliary physics or is it the same?

Reply:
Good question. I checked and there is no performance difference, but `if(boolean) return` is more easy to read and is more explicit, so I'll put it like that. I'll make it uniform though.

Reply:
here speed is of no concern TBH